Packs/EDL/Integrations/EDL/README.md

```diff
@@ -72,7 +72,16 @@ These parameters prevent the integration from incorrectly inserting unwanted TLD
 The default value for `Maximum CIDR network prefix bits size` is 8, which means that CIDRs with a lower network prefix bits number are not included (such as 0.0.0.0/2).
 
 
-The default value for `Exclude top level domainGlobs` is off. If enabled, the exported list does not hold indicators such as \*.com, \*.co.uk, \*.org and other top level domains.
+The default value for `Exclude top level domainGlobs` is off. If enabled, the exported list does not hold indicators such as `*.com`, `*.co.uk`, `*.org` and other top level domains.
+
+### DomainGlobs
```

Comment:
i'm not sure if we should name the section like this.
maybe 'Behaviors' or something like that.
@Ni-Knight wdyt? so it's safeguards and then behaviors..?

<img width="276" alt="image" src="https://user-images.githubusercontent.com/73779436/165737709-a61b5831-fe1c-48df-a88e-ca48973444be.png">

Reply:
maybe?
### Unique Behaviors
#### DomainGlobs
Text Text Text
#### IP Ranges
Text Text Text
```
### Unique Behaviors
#### DomainGlobs
Text Text Text
#### IP Ranges
Text Text Text
```

Reply:
The `IP Ranges` and the `Drop Invalids` are already documented in the parameters/arguments section